### PR TITLE
ROX-23972: Fix Operator Network descriptions in UI

### PR DESCRIPTION
**Pull request by ebensh**
### Description

The work done in ROX-23972 made the UI look weird; this tries to fix some wording so it looks better.

Before:
Screenshot from 2024-07-04 09-00-52
Screenshot from 2024-07-04 08-59-05

### User-facing documentation

- [x] CHANGELOG update is not needed
- [x] Documentation is not needed

### Testing

- [x] inspected CI results

#### Automated testing

- [x] contributed **no automated tests**

#### How I validated my change

TODO - verifying with Marcin best way to validate

### CODE REVIEW

File: operator/apis/platform/v1alpha1/central_types.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ type CentralSpec struct {
 	Monitoring *GlobalMonitoring `json:"monitoring,omitempty"`
 
 	// Network configuration.
-	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=10,xDescription={"urn:alm:descriptor:com.tectonic.ui:advanced"}
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName=Network,order=10,xDescription={"urn:alm:descriptor:com.tectonic.ui:advanced"}
```
Comment (ebensh) [inline operator/apis/platform/v1alpha1/central_types.go:76]: Capitalized from network to Network section header

File: operator/apis/platform/v1alpha1/common_types.go [hunk 2]
```diff
@@ -349,7 +349,8 @@ func (s *GlobalNetworkSpec) IsNetworkPoliciesEnabled() bool {
 // GlobalNetworkSpec defines settings related to Helm chart network parameters. The corresponding Helm flags
 // live in the global scope `.network`.
 type GlobalNetworkSpec struct {
+	// Create NetworkPolicy objects to limit ACS network connectivity.
 	//+kubebuilder:default=Enabled
-	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=1
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=1,displayName="Network Policies"
```
Comment (ebensh) [inline operator/apis/platform/v1alpha1/common_types.go:354]: Changed display from "policies" to "Network Policies"

File: operator/apis/platform/v1alpha1/common_types.go [hunk 3]
```diff
@@ -349,7 +349,8 @@ func (s *GlobalNetworkSpec) IsNetworkPoliciesEnabled() bool {
 // GlobalNetworkSpec defines settings related to Helm chart network parameters. The corresponding Helm flags
 // live in the global scope `.network`.
 type GlobalNetworkSpec struct {
+	// Create NetworkPolicy objects to limit ACS network connectivity.
```
Comment (ebensh) [inline operator/apis/platform/v1alpha1/common_types.go:352]: Added comment to clarify the setting's behavior.

File: operator/apis/platform/v1alpha1/central_types.go (hunk 1 above)
Comment (porridge) [inline operator/apis/platform/v1alpha1/central_types.go:76]: This is likely why it wasn't hidden:
```suggestion
	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName=Network,order=10,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
```

**Review by porridge — CHANGES_REQUESTED** [review]

File: operator/apis/platform/v1alpha1/central_types.go (hunk 1 above)
Comment (ebensh) [reply operator/apis/platform/v1alpha1/central_types.go]: Good catch, thank you! Will copy-paste next time

**Review by vladbologa — CHANGES_REQUESTED** [review]
There's the `xDescription` typo also in `securedcluster_types.go`

**Comment by ebensh** [comment]
> There's the `xDescription` typo also in `securedcluster_types.go`

Fixed, thanks

**Comment by porridge** [comment]
There are "before" screenshots, but I cannot find the "after"?

**Review by porridge — APPROVED** [review]

**Comment by porridge** [comment]
Approving since it's clearly an improvement, but it would be good to see the difference in UI.

**Pull request merged**